doc/user/content/sql/functions/to_char.md

```diff
@@ -0,0 +1,110 @@
+---
+title: "to_char Function"
+description: "Converts a timestamp into a string using the specified format."
+menu:
+  main:
+    parent: 'sql-functions'
+---
+
+`to_char` returns a string created using the specified format string.
+
+The format string can be composed of any number of format tokens, interspersed with
```

Comment:
These are referred to as "format tokens" here but "format specifiers" below; let's make those consistent and introduce links between this reference and the header where they can be found.